src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/ConditionPatternParser.cs

```diff
@@ -139,7 +138,7 @@ public static ParsedModRewriteInput ParseActionCondition(string condition)
         /// <param name="context"></param>
         /// <param name="invert"></param>
         /// <returns></returns>
-        public static ParsedModRewriteInput ParseProperty(ParserContext context, bool invert)
+        private ParsedModRewriteInput ParseProperty(ParserContext context, bool invert)
```

Comment:
This only runs once on startup, would like to create the instances and have gc clean them up after use.